### PR TITLE
Add upgrade known issues for 1.3.1

### PR DESCRIPTION
**Pull request by bk201**
- https://github.com/harvester/harvester/issues/6041

Depends on https://github.com/harvester/docs/pull/604

### CODE REVIEW

**Review by innobead — APPROVED** [review]

File: v1.3.1.md [hunk 1]
```diff
@@ -58,6 +58,9 @@ Harvester only allows upgrades from supported versions. For more information abo
 > [!NOTE]
 > You can upgrade to v1.3.1 from both v1.3.0 and v1.2.2. If you are using an earlier version, you must upgrade to v1.2.2 before upgrading to v1.3.1.
 
+> [!WARNING]
+> Please check https://docs.harvesterhci.io/v1.3/upgrade/v1-2-2-to-v1-3-1 for known issues. There are some reports from the community about issue [#6041](https://github.com/harvester/harvester/issues/6041). Please check the [Workaround](https://docs.harvesterhci.io/v1.3/upgrade/v1-2-2-to-v1-3-1#1-upgrade-stuck-after-the-first-node-upgraded) for how to resume the upgrade.
```
Comment (jillian-maroket) [inline v1.3.1.md:62]:
```suggestion
> [!IMPORTANT]
> You can upgrade to v1.3.1 from both v1.3.0 and v1.2.2. If you are using an earlier version, you must upgrade to v1.2.2 before upgrading to v1.3.1.
> 
> The [documentation](https://docs.harvesterhci.io/v1.3/upgrade/v1-2-2-to-v1-3-1) contains information about upgrade-related issues. If your cluster is affected by [issue #6041](https://github.com/harvester/harvester/issues/6041), check the suggested [workaround](https://docs.harvesterhci.io/v1.3/upgrade/v1-2-2-to-v1-3-1#1-upgrade-stuck-after-the-first-node-upgraded) in the documentation.
```

Comment (jillian-maroket) [reply v1.3.1.md:62]: I suggest combining the notes to avoid cluttering the release notes. I think "Important" is more appropriate for the new text.

**Review by jillian-maroket — COMMENTED** [review]
PTAL at the suggested changes

File: v1.3.1.md (hunk 1 above)
Comment (bk201) [reply v1.3.1.md]: Thank you! Merge the change.

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**